### ISSUE TITLE
No validation for uninitialized ensembles in manual update/evaluate experiment

### ISSUE THREAD

**Issue by larsevj** [equinor/ert#8691]
### What happened? (You can include a screenshot if it helps explain)

Open ert gui and create a new experiment / ensemble in the manage experiments window, then attempt  to run either manual update or evaluate ensemble. 

<img width="1195" alt="image" src="https://github.com/user-attachments/assets/c0ff3905-19dc-4577-9b58-6ad446f4df14">

### What did you expect to happen?

Should validate and disable run experiment button.

### steps to reproduce

```Python
ert gui poly.ert
Create new experiment / ensemble
Run either manual update or evaluate ensemble
```

### Environment where bug has been observed

- [ ] python 3.8
- [ ] python 3.9
- [ ] python 3.10
- [X] python 3.11
- [ ] python 3.12
- [X] macosx
- [ ] rhel7
- [ ] rhel8
- [X] local queue
- [ ] lsf queue
- [ ] slurm queue
- [ ] openPBS queue
- [ ] used flag `--enable-scheduler`

**Comment by jonathan-eq** [comment]
The button seems to already be disabled. ![Image](https://github.com/user-attachments/assets/bde0bf60-9b1a-4aaa-85f1-a00c6e5d3a5f)

**Comment by eivindjahren** [comment]
Just wanted to add that you get the following by running `load_results_manually` when runpath hasn't been created:

![Image](https://github.com/user-attachments/assets/0f166f08-f237-454c-bad3-84f37cdda702)

**Comment by jonathan-eq** [comment]
> Just wanted to add that you get the following by running `load_results_manually` when runpath hasn't been created:
> 
> ![Image](https://github.com/user-attachments/assets/0f166f08-f237-454c-bad3-84f37cdda702)

Resolved by #8971
